### PR TITLE
Case 20550 - install Nitpick on windows

### PR DESCRIPTION
**Pull request by NissimHadar**
https://highfidelity.fogbugz.com/f/cases/20550/Add-nitpick-to-HiFi-installer

# Description
nitpick is now installed by the High Fidelity installer on Windows.
It is located next to the server-console binaries
# Testing
1.  Download and install the PR at the default location
2.  Verify that `C:\Program Files\High Fidelity - PR14693` contains a folder named `nitpick`
3.  Open that folder - `C:\Program Files\High Fidelity - PR14693\nitpick`
4.  Run `nitpick.exe` and verify that the application opens.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16386/HighFidelity-Beta-PR14693-524a8e5.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16386/HighFidelity-Beta-PR14693-524a8e5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16386/HighFidelity-Beta-Interface-PR14693-524a8e5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16386/HighFidelity-Beta-PR14693-524a8e5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16386/HighFidelity-Beta-Interface-PR14693-524a8e5.exe

File: tools/nitpick/CMakeLists.txt [hunk 1]
```diff
@@ -73,3 +73,20 @@ elseif (APPLE)
...
+
+  install(
+    DIRECTORY "${NITPICK_DESTINATION}/"
+    DESTINATION ${NITPICK_INSTALL_DIR}
+  )
+
+  # sign the copied server console executable after install
+  set(EXECUTABLE_PATH "${NITPICK_DESTINATION}/${NITPICK_EXEC_NAME}")
+  optional_win_executable_signing()
+endif()
```
Comment (AndrewMeadows) [inline tools/nitpick/CMakeLists.txt]: Why not combine these two `if` blocks?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16490/HighFidelity-Beta-PR14693-3cc225f.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16490/HighFidelity-Beta-PR14693-3cc225f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16490/HighFidelity-Beta-Interface-PR14693-3cc225f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16490/HighFidelity-Beta-PR14693-3cc225f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16490/HighFidelity-Beta-Interface-PR14693-3cc225f.exe

**Comment by qaAdamSmith82** [comment]
Currently Testing

**Comment by qaAdamSmith82** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16647/HighFidelity-Beta-PR14693-3cc225f.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16647/HighFidelity-Beta-PR14693-3cc225f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16647/HighFidelity-Beta-Interface-PR14693-3cc225f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16647/HighFidelity-Beta-PR14693-3cc225f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16647/HighFidelity-Beta-Interface-PR14693-3cc225f.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16776/HighFidelity-Beta-PR14693-a5fcfcc.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16776/HighFidelity-Beta-PR14693-a5fcfcc.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16776/HighFidelity-Beta-Interface-PR14693-a5fcfcc.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16776/HighFidelity-Beta-PR14693-a5fcfcc.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16776/HighFidelity-Beta-Interface-PR14693-a5fcfcc.exe

**Pull request closed without merging**